examples/XX-anomaly-detection.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<p>fittable scorer --&gt; fittable detector.</p><p> </p><p>"It requires a parameter <code>high_quantile</code> that indicates the fraction of points considered normal" --&gt; technically this is determined by the combination of <code>high_quantile</code> and <code>low_quantile</code>.</p><p> </p><p>"The detector will flag all timestamps with an anomaly score higher than the 94% quantile. Therefore, the parameter needs to be chosen carefully" --&gt; The "therefore" sounds weird here. It is not the fact that you use the detector with 0.94 that implies that the parameter has to be chosen carefully.</p><p> </p><p>"We are expected to have eight anomalous days, which translates to an estimated 94% of normal days in our dataset. " --&gt; Why 8 ? I thought there were 5 anomalous days. Also: be careful with this, because it's a form of data leakage from the test set (the number of anomalous days is not known in advance). I would feel more confortable if you were using a value such as 90%, 95% or 99% without such a justification, or be really clear that this is just for the sake of the example and in practice the optimal ratio would have to be found.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/1518/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>